### PR TITLE
Fix links for mobile

### PR DESCRIPTION
**Pull request by 0bman**

### CODE REVIEW

File: app/assets/javascripts/mobile.coffee [hunk 1]
```diff
@@ -2,6 +2,11 @@
 #= require _swipe
 
 [].forEach.call document.querySelectorAll('nav a'), (a) ->
+  currentUrl = document.location.pathname
+  url = a.getAttribute('href')
+  isRootUrl = url == 'home' && currentUrl == '/'
+  a.className += 'active' if "/#{url}" == currentUrl || isRootUrl
+
```
Comment (galulex) [inline app/assets/javascripts/mobile.coffee]:
```javascript
a.className = ''
```

```javascript
  document.querySelector("[href='" + location.pathname.replace('/', '') + "']").setAttribute('class', 'active')
```

File: app/assets/javascripts/_swipe.js [hunk 2]
```diff
@@ -1,10 +1,12 @@
-var startX = startY = endX = endY = swipeX = swipeY = page = 0,
-  pages = ['home', 'about', 'services', 'testimonials', 'contact'],
+var pages = ['home', 'about', 'services', 'testimonials', 'contact'],
+  startX = startY = endX = endY = swipeX = swipeY = page = pages.indexOf(getCurrentUrl()),
```
Comment (galulex) [inline app/assets/javascripts/_swipe.js]:
```javascript
currentPage = document.location.pathname.replace('/', '')
startX = startY = endX = endY = swipeX = swipeY = page = pages.indexOf(currentPage) || 0,
```

File: app/assets/javascripts/mobile.coffee [hunk 3]
```diff
@@ -2,6 +2,8 @@
 #= require _swipe
 
 [].forEach.call document.querySelectorAll('nav a'), (a) ->
+  a.className = ''
+  document.querySelector("nav a[href='" + location.pathname + "']").setAttribute('class', 'active')
```
Comment (galulex) [inline app/assets/javascripts/mobile.coffee]: We should not place it here remove this line from the loop. We need to run it only once

Comment (0bman) [inline app/assets/javascripts/mobile.coffee]: done

**Pull request merged**